frontend/lib/identification/card_detail_view/extend_card_notification.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:flutter/material.dart';
+
+import 'package:ehrenamtskarte/build_config/build_config.dart' show buildConfig;
+import 'package:ehrenamtskarte/l10n/translations.g.dart';
+import 'package:url_launcher/url_launcher_string.dart';
+
+class ExtendCardNotification extends StatefulWidget {
+  @override
+  State<ExtendCardNotification> createState() => _ExtendCardNotificationState();
+}
+
+class _ExtendCardNotificationState extends State<ExtendCardNotification> {
+  bool _isVisible = true;
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context).colorScheme;
+    final t = context.t;
+
+    return _isVisible
+        ? Padding(
+            padding: const EdgeInsets.all(8.0),
+            child: Container(
+              padding: const EdgeInsets.all(16),
+              decoration: BoxDecoration(
+                color: theme.primary.withOpacity(0.1),
+                borderRadius: BorderRadius.circular(8),
+              ),
+              child: Row(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  Icon(Icons.info, color: theme.primary),
+                  SizedBox(width: 16),
+                  Expanded(
+                    child: Column(
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      children: [
+                        Text(t.identification.extendCardNotificationTitle),
+                        SizedBox(height: 8),
+                        Text(
+                          t.identification.extendCardNotificationDescription,
+                          textScaler: TextScaler.linear(0.9),
+                        ),
+                        SizedBox(height: 8),
```

Comment:
Is there a reason why you don't use the `Padding` widgets for set padding on texts and buttons?

Reply:
I wondered about that, but couldn't find any reason why Padding should be prefered other than in case of personal preference.
https://stackoverflow.com/questions/50722011/sized-box-vs-padding-for-distances-in-columns-and-rows
I personally think, the SizedBox looks neater, because the Padding implies a nested structure.
It's also used in other widgets already.

Reply:
I wondered about that, but couldn't find any reason why Padding should be prefered other than in case of personal preference. 
If you mean using Padding as a wrapper - then I think SizedBox would be neater because it doesn't imply a nested structure.
https://stackoverflow.com/questions/50722011/sized-box-vs-padding-for-distances-in-columns-and-rows
We also use the SizedBox in other widgets.

Reply:
Ok I'm fine with that even I think it's common use to add a padding around texts and is for me easier to understand the boilerplate Syntax since it's also common Syntax for every frontend dev.
Even you could also create a div Container as a spacer between components

But yes it may be personal preference